doc/install.md

```diff
@@ -20,8 +21,29 @@ If you do not have a cluster yet, create one by using one of the following provi
 * [kubeadm](https://kubernetes.io/docs/admin/kubeadm/)
 * [minikube](https://minikube.sigs.k8s.io/)
 
-## Install release v0.19.8 and use Coscheduling
-
+## Install release v0.19.8 as a second scheduler and use Coscheduling
+The quickest way to try scheduler-plugins is to install it using helm chart as a second scheduler.
+You can find the demo chart in [manifests/install/charts](../manifests/install/charts).
+
+1. Helm install.
+```bash
+$ git clone git@github.com:kubernetes-sigs/scheduler-plugins.git
+$ cd scheduler-plugins/manifests/install/charts
+$ helm install scheduler-plugins as-a-second-scheduler/
+```
```

Comment:
Make it indented:

```
    ```bash
    $ git clone git@github.com:kubernetes-sigs/scheduler-plugins.git
    $ cd scheduler-plugins/manifests/install/charts
    $ helm install scheduler-plugins as-a-second-scheduler/
    ```
```